src/Compute/Compute/ChangeLog.md

```diff
@@ -19,7 +19,10 @@
         - Additional information about change #1
 
 -->
-## Upcoming Release
+## Version 10.0.0
+* Deprecated `Get-AzVMSize` 'List Virtual Machine Size' parameter set. 
```

Comment:
Please do not update the version manually. Just update the change log under `Upcoming Release`. Version will be updated automatically when release.